### DISCUSSION TITLE
Backtesting.py Video Tutorials

### DISCUSSION THREAD

**Discussion by eervin123** [kernc/backtesting.py#677] [General]
I didn't see a `resources` category so am just throwing this here in general. I haven't contributed in a while to the community and feel bad about that because we use backtesting.py a lot. 

While I didn't make them, I thought I could at least compile a list here of helpful resources that I've come across in my travels.

Chad Thackray has a great youtube channel and also blogs his various tutorials. His great Scottish accent makes his videos so easy to watch and stay engaged. 

* Backtest Your Dollar Cost Average Strategy 
   * Youtube https://www.youtube.com/watch?v=Kgr8JjWX0zk
   * Here is the blog https://greyhoundanalytics.com/blog/backtest-your-dollar-cost-average-strategy-easily-in-python/
* Complete Backtesting.py tutorial
  * Blog Article https://greyhoundanalytics.com/blog/backtestingpy-a-complete-quickstart-guide/
  * Here is the youtube https://www.youtube.com/watch?v=e4ytbIm2Xg0&list=PLnSVMZC68_e4tQEW3thbstHjy-J55DDF8&index=20

AlgoVibes also does a great job on his tutorials. Straightforward coding with a nice easygoing style. 
* Python Backtesting Library you should DEFINITELY check out - Backtesting.py
  * Youtube link https://www.youtube.com/watch?v=FpSopSupizo

Martin Mayer Video
* Backtest a Moving Average Crossover Strategy with Intraday Prices in Python! Algorithmic Trading
  * Youtube https://www.youtube.com/watch?v=06FkYwHqsi4
  
  
I'll come back and post more here as I find them. Sorry @kernc  I know I said I would try to take the time to make some videos, but never got around to it. Figured the least I could do was share someone else's

**Comment by kernc** [comment]
@eervin123 This is great! 🤩 I wasn't aware there were videos covering the lib, and with such warmly dispositions. Thanks!

**Reply by eervin123** [reply]
Yeah it made me smile for you. It's funny, I couldn't wait to share them with you. 

You and the lib deserve the praise. 👏

**Reply by sultank1** [reply]
You are right..... 
 he is hero...
